modules/oe_showcase_page/oe_showcase_page.post_update.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * @file
+ * OpenEuropa Showcase Page post updates.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\field\Entity\FieldConfig;
+
+/**
+ * Enable the social_media paragraph.
+ */
+function oe_showcase_page_post_update_00001(&$sandbox) {
```

Comment:
```suggestion
function oe_showcase_page_post_update_00001(&$sandbox): void {
```
we dont specify return type only if its mixed, and only as long as we support php smaller than 8